coding-test/11일차/오픈채팅방.js

```diff
@@ -0,0 +1,50 @@
+function solution(record) {
+    //로직구성
+    //0. 받아온 문자열을 쪼개서 배열로 만든다.
+    //1. 객체로 새로 들어온 아이디와, 닉네임을 매치
+    //2. Enter, Leave, Change 에 따라서 닉네밍임이 변경되는 함수로 구현
+    //3. 
+    
+    const chatRoom = {};
+    const msg = [];
+    //받아온 user을 배열에 담는 함수
+    function create_user() {
+        const arr = [];
+        for(let i = 0; i < record.length; i++) {
+            arr[i] = record[i].split(' ');
+        }
+        return arr;
```

Comment:
```suggestion 
return recordList.map((record) => record.split(" "));
```
javascript의 배열 메서드 중 map이라는 메서드가 있습니다. 한번 찾아보심이 좋을것 같아요